staff/groups/aleti/project-app/src/logic/MovieApi.js

```diff
@@ -0,0 +1,39 @@
+const MovieApi = {
+
+    url: 'http://api.themoviedb.org/3',
+    key: 'c9e81d7384a0e7aa9d0deecb8c80c2cc',
+
+    
+    getTvPopular(){
+        return Promise.resolve()
+        .then(()=>{
+
+            return  fetch(`${this.url}/tv/popular?api_key=${this.key}&language=en-US&page=1`)
+            .then(resp => resp.json())
+            .then(info => console.log(info))
+        })
+    },
+    
+    getMoviesPopular(){
```

Comment:
TIP getPopular would be enough, instead of getMoviesPopular ("Movies" seems redundant, as api itself is already about movies)

Reply:
api is about movies and tv series so I have two methods: getMoviesPopular()  and getTvPopular() couse i need to discriminate each other